auth/postgres/key.go

```diff
@@ -0,0 +1,110 @@
+package postgres
+
+import (
+	"context"
+	"database/sql"
+	"time"
+
+	// required for DB access
+	"github.com/lib/pq"
+	"github.com/mainflux/mainflux/auth"
+)
+
+var _ auth.KeyRepository = (*keyRepository)(nil)
+
+const (
+	errDuplicate = "unique_violation"
+	errInvalid   = "invalid_text_representation"
+)
+
+type keyRepository struct {
+	db Database
+}
+
+// New instantiates a PostgreSQL implementation of key
+// repository.
```

Comment:
No need breaking this comment in two lines, especially that lines of code below are longer